myproject.template.yml

```diff
@@ -2,10 +2,26 @@
 # This is optional if this file is placed in the project directory under `.serena/project.yml`.
 project_root: /path/to/project
 
-# language of the project (csharp, python, rust, java, typescript, javascript, go, or ruby)
+# Default language of the project (csharp, python, rust, java, typescript, javascript, go, or ruby).
+# This is used to start the initial language server when the project is activated.
+#
+# If a tool operation targets a file with a different supported language (based on its extension),
+# Serena will attempt to restart the language server for that specific language.
+# If the file's language is not supported or cannot be determined, this default language server will
+# be used (if running).
+#
 # Special requirements:
 #  * csharp: Requires the presence of a .sln file in the project folder.
-language: python
+default_language: python
```

Comment:
How about instead we support an entry `languages` that can also be empty. If multiple are provided, then multiple LS instances are spun up.